### PR TITLE
[QA-AUTO] Create automate "board header" component tests

### PR DESCRIPTION
**Pull request by annaslodkowska**

### CODE REVIEW

**Review by KrzysztofGajQA — DISMISSED** [review]
lgtm

File: src/test/java/com/intive/template/pages/BoardHeaderPage.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
+package com.intive.template.pages;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.How;
+
+
```
Comment (ospatrycja) [inline src/test/java/com/intive/template/pages/BoardHeaderPage.java:7]: Minor thing: one empty line too much

File: src/test/java/com/intive/template/pages/BoardHeaderPage.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    @FindBy(how = How.XPATH, using = "/html/body/div/div/section/a")
+    WebElement returnToMainBoardLink;
+
+    @FindBy(how = How.XPATH, using = "/html/body/div/div/section/div/h1")
+    WebElement title;
+
+    @FindBy(how = How.XPATH, using = "/html/body/div/div/section/div/button")
+    WebElement newColumnButton;
+
+
```
Comment (ospatrycja) [inline src/test/java/com/intive/template/pages/BoardHeaderPage.java:19]: Minor thing: one empty line too much

File: src/test/resources/features/boardHeaderComponent.feature [hunk 3]
```diff
@@ -0,0 +1,10 @@
+Feature: Board header component feature
```
Comment (ospatrycja) [inline src/test/resources/features/boardHeaderComponent.feature:1]: Could you add tag number of ticket with tests cases? According to Marek's request.

**Review by ospatrycja — APPROVED** [review]
Apart from some minor formatting things and missing ticket tag number looks good to me

File: src/test/java/com/intive/template/pages/BoardHeaderPage.java (hunk 1 above)
Comment (MarekPawlowskiAuto) [reply src/test/java/com/intive/template/pages/BoardHeaderPage.java:7]: Remember to use IntelliJ default formatting cmd + option + shift + L (or equivalent on windows/Linux)

Comment (annaslodkowska) [reply src/test/java/com/intive/template/pages/BoardHeaderPage.java]: Thanks for tips. Done.

File: src/test/java/com/intive/template/pages/BoardHeaderPage.java (hunk 2 above)
Comment (annaslodkowska) [reply src/test/java/com/intive/template/pages/BoardHeaderPage.java]: Thanks for finding. Done.

File: src/test/resources/features/boardHeaderComponent.feature (hunk 3 above)
Comment (annaslodkowska) [reply src/test/resources/features/boardHeaderComponent.feature:1]: Thanks for your suggestion. Done.

**Comment by annaslodkowska** [comment]
@KrzysztofGajQA @ospatrycja @MarekPawlowskiQA Thanks for your review, all findings fixed. Ready for re-review.

**Review by MarekPawlowskiAuto — APPROVED** [review]
lgtm :)

**Review by MarekPawlowskiQA — APPROVED** [review]

**Review by ospatrycja — APPROVED** [review]
Looks good to me :)

**Pull request merged**